server/config.go

```diff
@@ -427,8 +442,50 @@ func (o *scheduleOption) GetSchedulers() SchedulerConfigs {
 	return o.load().Schedulers
 }
 
-func (o *scheduleOption) GetMeta() toml.MetaData {
-	return o.load().Meta
+func (o *scheduleOption) AddSchedulerCfg(tp string, args []string) bool {
+	stringSliceEqual := func(a, b []string) bool {
+		if len(a) != len(b) {
+			return false
+		}
+		if (a == nil) != (b == nil) {
+			return false
+		}
+		for i, v := range a {
+			if v != b[i] {
+				return false
+			}
+		}
+		return true
+	}
+
+	c := o.load()
+	v := c.clone()
+	for _, schedulerCfg := range v.Schedulers {
+		// comparing args is to cover the case that there are schedulers in same type but not with same name
+		// such as two schedulers of type evictLeader,
+		// one name is "evict-leader-scheduler-1" and the other is "evict-leader-scheduler-2"
+		if schedulerCfg.Type == tp && stringSliceEqual(schedulerCfg.Args, args) {
+			return false
+		}
```

Comment:
How about add a `Equals` method to `SchedulerConfig`?

Reply:
OK